glass/shells.py

```diff
@@ -54,24 +53,31 @@
 from glass.core.array import ndinterp
 
 if typing.TYPE_CHECKING:
-    from cosmology import Cosmology
+    import collections.abc
 
-# types
-ArrayLike1D = typing.Union[collections.abc.Sequence[float], npt.NDArray[typing.Any]]
-WeightFunc = typing.Callable[[ArrayLike1D], npt.NDArray[typing.Any]]
```

Comment:
I'm not sure eliminating this type alias makes things more readable -- to me, it makes it harder to understand the signatures below.

Reply:
Okay, your call. I'm not a fan of the custom types.

Reply:
Although things like `WeightFunc` currently don't all have the same signature. So will require some fiddling.